### ISSUE TITLE
Driver doesnt work in IDE's (sublime, pycharm,spider) but only works in cmd

### ISSUE THREAD

**Issue by koeliga** [ultrafunkamsterdam/undetected-chromedriver#421]
When running any code in sublime, pycharm or spyder I always get this message and the code stops.
Code I'm currently using: https://imgur.com/a/fNaUPu2

RuntimeError: 
        An attempt has been made to start a new process before the
        current process has finished its bootstrapping phase.

        This probably means that you are not using fork to start your
        child processes and you have forgotten to use the proper idiom
        in the main module:

            if __name__ == '__main__':
                freeze_support()
                ...

        The "freeze_support()" line can be omitted if the program
        is not going to be frozen to produce an executable..

When adding if __name__ == '__main__': in front of the code it crashes: https://imgur.com/a/YyUtWDS
But when I run the same code in python cmd it works just fine: https://imgur.com/a/ZYPUrJY

I have a 5 5600g apu and python 3.10 if that helps

Any help would be greatly appreciated, 
Thanks in advance

**Comment by ThePakeha** [comment]
Having the same issue, did you find a solution?

**Comment by koeliga** [comment]
> Having the same issue, did you find a solution?

@ThePakeha I did actually. I found out that the code is actually running it just quits when it has finished. So when you putt sleep after the code it shouldn't quit. Here is the code I'm running: `import undetected_chromedriver as uc
from time import sleep
if __name__ == '__main__':
    driver = uc.Chrome()
    driver.get('https://nowsecure.nl')
    sleep(10)`
Also make sure there are four spaces after the if__name.......: that might also be a problem.
Let me know if it helped
